index.html

```diff
@@ -0,0 +1,107 @@
+<div class="main">
+  <header id="header">
+    <div class="logo">
+      <img
+           id="header-img"
+           alt="paz logo"
+           src="https://www.logosoftwear.com/responsive/img/global/mobile-icon.png"
+    </div>
+    <nav id="nav-bar">
+      <ul>
+        <li><a class="nav-link" href="#features">Features</a></li>
+        <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
+        <li><a class="nav-link" href="#pricing">Pricing</a></li>
+      </ul>
+    </nav>     
+  </header>
+          <div class="form">
+        <h2>Handcrafted, home-mad masterpieces</h2>
+        <form id="form" action="https://wwww.freecodecamp.com/email-submit">
+          <input
+                 type="email"
+                 placeholder="Enter your email address"
+                 name="email"
+                 id="email"
+                 required
+              />
+          <input id="submit" type="submit" value="Get Started"/>
+        </form>
+      </div>
+    <div id="features"> 
+    <div class="card">
+      <div class="icon"><i class="fa fa-3x fa-fire"></i></div>
+        <div class="content">
+          <h2> Premium Materials</h2>
+            <p>
+              Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.
+            </p>
+        </div>     
+      </div>
+      
+      <div class="card">
+        <div class="icon"><i class="fa fa-3x fa-truck"></i></div>
+        <div class="content">
+          <h2>Fast Shipping</h2>
+          <p>
+            We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.
+          </p>
+        </div>        
+      </div>
```

Comment:
Is a good practice to use <article> tags instead of <div class="card">. What do you think if you change them?

Reply:
- [ ] Also here, I am not sure which is a good practice to use. Maybe it's something in the [HTML & CSS best practices guide](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)? Please, provide a better explanation of your suggestions :raised_hands: